oeps/best-practices/oep-0080-openfeature-feature-flags.rst

```diff
@@ -0,0 +1,66 @@
+.. _external_link_template:
+
+.. Below is the display in the left sidebar on RTD. Please omit leading 0's
+
+OEP-80: OpenFeature Feature Flags
+#################################
+
+.. This OEP template is recommended when content is externally linked.
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - Link to the doc in the following format::
+
+        :doc:`OEP-XXXX <oep-XXXX-YYYY-ZZZZ>`
+
+        * <XXXX is the next available OEP number>
+        * <YYYY is the abbreviated Type: proc | bp | arch>
+        * <ZZZZ is a brief (< 5 words) version of the title>
+
+   * - Title
+     - OpenFeature Feature Flags
+   * - Last Modified
+     - 2024-12-13
+   * - Authors
+     - Christ Patti <cpatti@mit.edu>, Tobias Macey <tmacey@mit.edu>
+   * - Arbiter
+     - <Arbiter's real name and email address>
+   * - Status
+     - <Draft
+   * - Type
+     - Best Practice
+   * - Created
+     - 2024-12-13
+   * - Review Period
+     - <start - target end dates for review>
+
+Abstract
+********
+
+Right now OpenEdX has a huge array of toggles and feature flags in a number of repositories and using a number of formats.
```

Comment:
Just a note that as you're writing this, please use the spelling/spacing/capitalization "Open edX" (so I don't have to comment a million times 😅 ) and use it as a noun, please! (eg "The Open edX platform")